### PR TITLE
Removed logo in fedora according to guidelines

### LINKED ISSUE #2007: Fedora pages don't follow logo usage guidelines
### Expected behaviour

Usage of the Fedora logo is in accordance with the logo usage guidelines https://fedoraproject.org/wiki/Logo/UsageGuidelines#Common_guidelines

### Steps to reproduce the problem

Go to https://snapcraft.io/install/spotify/fedora

### Specs

- _URL:_
- _Operating system:_
- _Browser:_

### PR DESCRIPTION
**Pull request by therealjuan**
Fixes #2007 

### QA Steps

1. Pull the branch or open the demo service
2. Open any distro pages with Fedora instructions like `/install/spotify/fedora`
3. Check the logo is not visible in the background according to [Fedora logo guidelines](https://fedoraproject.org/wiki/Logo/UsageGuidelines)
4. Visit another distro pages like openSuse `/install/spotify/opensuse`
5. Notice the logo should be there
6. #winning

### CODE REVIEW

File: webapp/store/content/distros/arch.yaml [hunk 1]
```diff
@@ -2,6 +2,7 @@ name: Arch Linux
 color-1: "#1793D1"
 color-2: "#116F9E"
 logo: https://assets.ubuntu.com/v1/feca0fc0-Distro_Logo_ArchLinux.svg
+display-logo: true
```
Comment (bartaz) [inline webapp/store/content/distros/arch.yaml]: Can't we make it true by default? And only set to no in fedora?

Also 'display-logo' is a bit misleading - logo is still shown on a page, just not in background.

**Comment by kwm14** [comment]
@therealjuan is it odd to remove/hide the logo for Fedora but keep it visible for other distro pages?

**Comment by bartaz** [comment]
Also - if we want to be picky - it seems our Ubuntu logo usage also is against out own guidelines: https://design.ubuntu.com/brand/ubuntu-logo/

Should we check all the distros and remove all logos? I guess most of them will have something against being shown on similar colour backgrounds.

**Comment by kwm14** [comment]
## Distro Brand Guidelines
Reviewed guidelines for each distro to see if we can manipulate logo to sit over a coloured background colour.

The votes are in ✌️ 

- Arch Linux - **[No](https://wiki.archlinux.org/index.php/DeveloperWiki:TrademarkPolicy#Logo_Usage_Guidelines)**
- CentOS - **[No](https://wiki.centos.org/ArtWork/Brand/Logo#head-f379d7da0b5da9a7161148c5c9a093a00085975b)**
- Debian - **[Undisclosed](https://www.debian.org/logos/#open-use)**
- elementary OS - **[No](https://elementary.io/brand)**
- Fedora - **[No](https://fedoraproject.org/wiki/Logo/UsageGuidelines#Never_Use_the_Logo_on_Similarly-Colored_Backgrounds)**
- KDE Neon - **[Undisclosed](https://community.kde.org/Promo/Guidance/Branding)**
- Kubuntu - **[Undisclosed](https://community.kde.org/Kubuntu/Artwork)**
- Manjaro - **[No guidelines](https://gitlab.manjaro.org/artwork/branding/logo
)**
- Linux Mint - **[No guidelines #redesign](https://blog.linuxmint.com/?p=3732)**
- OpenSUSE - **[No](https://en.opensuse.org/openSUSE:Artwork_brand#Things_to_Avoid)**
- Ubuntu **Yes** (Internal review with the brand squad)

**Comment by therealjuan** [comment]
I would vote for removing the logo completely from the background as easier to maintain...

**Comment by therealjuan** [comment]
Closing as it we decided to remove the logo from the background as it clashes with a big % of the logo usage guidelines across different distros.

**Comment by mattdm** [comment]
Please note that there is another issue — the logo in use for Fedora is a strange variant (that is, not _actually_ our logo). Please see https://fedoraproject.org/wiki/Logo#Can_I_get_this_logo_in_other_sizes_or_formats.3F to get the correct logo. Thanks!

**Comment by therealjuan** [comment]
Thanks @mattdm. We already got the right assets as a comment in here: https://github.com/canonical-web-and-design/snapcraft.io/issues/2007

**Comment by mattdm** [comment]
@therealjuan Awesome thanks (and of course thanks @mairin)

**Pull request closed without merging**